test/e2e/pageLevelProgress.cy.js

```diff
@@ -0,0 +1,41 @@
+describe('Page Level Progress', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/')
+  });
+  
+  it('should display the page level progress bars correctly on the menu items', function () {
+    const isPageLevelProgressEnabled = this.data.course._pageLevelProgress._isEnabled
+    // Check if PLP is enabled. Check it's visible on menu tiles
+    if (isPageLevelProgressEnabled) {
+      const pagesCount = this.data.contentObjects.filter((page) => page._pageLevelProgress._isEnabled === true).length
```

Comment:
The _isEnabled value is boolean, so we don't need triple equals here
```suggestion
      const pagesCount = this.data.contentObjects.filter((page) => page._pageLevelProgress._isEnabled).length
```